README.md

```diff
@@ -64,11 +64,11 @@ Then get the source and install it:
 
 Before working on the code, if you plan to contribute your changes, you need to
 read the
-[Chef Contributions document](http://docs.opscode.com/community_contributions.html).
+[Chef Contributions document](http://docs.chef.io/community_contributions.html).
 
 You will also need to set up the repository with the appropriate branches. We
 document the process on the
-[Working with Git](http://wiki.opscode.com/display/chef/Working+with+git) page
+[Working with Git](http://wiki.chef.io/display/chef/Working+with+git) page
```

Comment:
I bet that information was from 3 years and out of date anyway.  Something along the lines of "just make a pull request against master" is the current process for external contributors.